src/Tests/Microsoft.Diagnostics.Monitoring.TestCommon/ConsoleOutputHelper.cs

```diff
@@ -3,24 +3,33 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.IO;
 using Xunit.Abstractions;
 
 namespace Microsoft.Diagnostics.Monitoring.TestCommon
 {
     public sealed class ConsoleOutputHelper : ITestOutputHelper
     {
-        public ConsoleOutputHelper()
+        private readonly bool _stdout;
+
+        public ConsoleOutputHelper(bool stdout = true)
         {
+            _stdout = stdout;
```

Comment:
Could this store the `TextWriter` instance instead of constantly looking it up based on the `bool` field?

Reply:
Yeah, I think it's okay to cache that without disposing it.